Sources/Fuzzilli/Lifting/JavaScriptLifter.swift

```diff
@@ -187,7 +187,7 @@ public class JavaScriptLifter: Lifter {
                 output = ObjectLiteral.new("{" + properties.joined(separator: ",") + "}")
 
             case is CreateArray:
-                let elems = instr.inputs.map({ expr(for: $0).text }).joined(separator: ",")
+                let elems = instr.inputs.map({ expr(for: $0).text == "undefined" ? "" : expr(for: $0).text }).joined(separator: ",")
```

Comment:
Nit: can you please add a comment that (1) explains what this does (e.g. "treat undefined elements as holes") and (2) mention that this relies on literals always being inlined (otherwise, `expr(for: $0).text` would just give you something like `v3`)? Also maybe it would be slightly nicer to only evaluate `expr(for: $0).text` once, but then you'd need to assign the result to a temporary variable. Really up to you what you prefer :)

Reply:
done, comments and syntax cleanup added ;)